responsiveness-homework-starter-code/css/index.css

```diff
@@ -33,16 +35,38 @@ header, .experience {
 }
 .containertext
 {
-	width: 60%;
+
 	height: auto;
 	border: 1px solid #E13A63;
 	display: inline-block;
+
 }
-.containerimg
+
+
+.containertext1
 {
-	width: auto;
 	height: auto;
 	border: 1px solid #E13A63;
 	display: inline-block;
 
 }
+
+
+.containerimg
+{
```

Comment:
Be consistent about how you write your code. Here, you write your css like this

.class-name
{
  attribute: value;
}

Whereas above, you write it like this:

.class-name {
  attribute: value;
}

Both are fine and there's not a huge difference, but it's distracting if things aren't consistent.